SQL/updates/9-10.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `ban` ADD `challenge` BOOLEAN NOT NULL DEFAULT FALSE AFTER `unbanned_ip`;
```

Comment:
This needs a comment and should really be multiline, for the sake of consistency